plugins/hls-alternate-number-format-plugin/test/testdata/TIntDtoND.expected.hs

```diff
@@ -2,4 +2,4 @@
 module TIntDtoND where
 
 convertMe :: Integer
-convertMe = 125.345e3
```

Comment:
Did you mean to change this?

Reply:
Yeah, I updated the way Integers are converted into NumDecimal Format